bootstrap/api/transport.go

```diff
@@ -16,6 +16,7 @@ import (
 	"github.com/go-zoo/bone"
 	"github.com/mainflux/mainflux"
 	"github.com/mainflux/mainflux/bootstrap"
+	internalerr "github.com/mainflux/mainflux/internal/errors"
```

Comment:
I would rather move errors in `pkg` so people can import it and use it on their Mainflux Apps (Apps on top of Mainflux, competing errors, etc...). It's something that I see as usable for sharing, not for internal usage only.

Reply:
@drasko @nmarcetic I'm not sure about this one. We have our mainflux/errors package there and the alias wouldn't be `internalerr` but probably `pkgerr`. Any suggestion?

Reply:
Yea IMHO `pkgErr` sounds good. Why don't you move it there then? in `pkg/errros` put a new file like `pkt/errors/types.go` or something like that so those errors that you are sharing will be part of our `pkg/errors` package. 
Even if you leave everything as you initially did, in `internal/errors` IMHO it doesn't make sense if we already have shared `pkg/errors` , merge it. What do you think @manuio  ?